src/pages/AboutUs.tsx

```diff
@@ -10,41 +10,57 @@ import julien from '../assets/images/julien.png';
 
 const AboutUs: React.FC<RouteComponentProps> = () => (
   <Box>
-    <Box px={16} pt={4} pb={6}>
-      <Typography
-        variant="h3"
-        align="center"
-        style={{ marginTop: '16px', marginBottom: '8px' }}
-      >
-        Our Team
-      </Typography>
-      <Typography variant="body1" align="center">
-        Apps like Trade Republic and other direct brokers make investing easier
-        and easier. The stock market is now accessible to everyone. Even so,
-        many individuals do not have access to tools to make informed and
-        informed investment decisions. Current solutions are often confusing and
-        tailored to professional investors. This is exactly where we attack.
-      </Typography>
+    <Box pt={4} pb={6}>
+      <Grid container direction="column" alignItems="center">
+        <Grid item>
+          <Typography
+            variant="h3"
+            align="center"
+            style={{ marginTop: '16px', marginBottom: '16px' }}
+          >
+            Our Team
+          </Typography>
+        </Grid>
+        <Grid item>
+          <Typography
+            variant="body1"
+            align="center"
+            style={{ maxWidth: '1200px' }}
+          >
+            Apps like Trade Republic and other direct brokers make investing
+            easier and easier. The stock market is now accessible to everyone.
+            Even so, many individuals do not have access to tools to make
+            informed and informed investment decisions. Current solutions are
+            often confusing and tailored to professional investors. This is
+            exactly where we attack.
+          </Typography>
+        </Grid>
+      </Grid>
     </Box>
     <Box
       bgcolor="#0c1a3a"
       color="white"
       style={{
-        padding: '80px',
+        padding: '100px',
```

Comment:
Refactor to use the [p property of the Box component ](https://material-ui.com/system/spacing/#api)